lib/glific_web/flows/flow_editor_controller.ex

```diff
@@ -521,6 +522,31 @@ defmodule GlificWeb.Flows.FlowEditorController do
     json(conn, res)
   end
 
+  @doc false
+  @spec sheets(Plug.Conn.t(), nil | maybe_improper_list | map) :: Plug.Conn.t()
+  def sheets(conn, _params) do
+    results =
+      Sheets.list_sheets(%{
+        filter: %{organization_id: conn.assigns[:organization_id]}
+      })
+      |> Enum.reduce([], fn sheet, acc ->
+        [
+          %{
+            id: sheet.id,
+            name: sheet.label,
+            url: sheet.url,
+            is_active: sheet.is_active,
```

Comment:
Why do we need these fields for the flow editor?
            last_synced_at: sheet.last_synced_at,
            created_on: sheet.inserted_at,
            modified_on: sheet.updated_at
            is_active: sheet.is_active
            
Also, I don't think we should send inactive sheets to the flow editor listing. It will create a lot of confusion.